extensions/gdx-setup/src/com/badlogic/gdx/setup/GdxSetupUI.java

```diff
@@ -648,7 +654,7 @@ public void actionPerformed (ActionEvent e) {
 			sdkLocationButton.addActionListener(new ActionListener() {
 				@Override
 				public void actionPerformed (ActionEvent e) {
-					File path = getDirectory();
+					File path = getDirectory("Choose Android SDK");
```

Comment:
It took me a second to notice this, but this is a great change. The previous code should not have used "Choose destination" for the Android SDK, good catch.